### ISSUE TITLE
Integration with nvim-cmp

### ISSUE THREAD

**Issue by VonHeikemen** [VonHeikemen/fine-cmdline.nvim#7]
I'm afraid that's not possible right now. The problem is [cmp-cmdline](https://github.com/hrsh7th/cmp-cmdline) only works in `commandline-mode`.

The solution to this is to make `cmp-cmdline` work in a normal buffer. So... you (not me) would need to create a new `nvim-cmp` source that has the same features as `cmp-cmdline`.

Let's imagine you already created this new source, here is what you'll to do to make it work (in theory).

```lua
local cmp = require('cmp')

require('fine-cmdline').setup({
  hooks = {
    after_mount = function(input)
      cmp.setup.buffer({
        sources = {
          {name = 'new-imaginary-source'}
        }
      })
    end
  }
})
```

**Comment by hemedani** [comment]
It's a very important feature

**Comment by mkarbo** [comment]
So... ?

**Comment by VonHeikemen** [comment]
> So... Any updates on this? wave smiley

Nope. No one has taken the time to create the source for `nvim-cmp` that it's needed.
